database_cleanup/tests/test_purge_fields.py

```diff
@@ -54,3 +54,13 @@ def test_empty_field(self):
                     ]
                 )
             )
+
+    @classmethod
+    def tearDownClass(self):
+        super().tearDownClass()
+        with environment() as env:
+            model = env["ir.model"].search(
+                [("name", "=", "Database cleanup test field-model")]
+            )
+            if model:
+                model.unlink()
```

Comment:
Curiosity: why this is now needed? Any error or warning raised by Odoo regarding this data model?

Otherwise I would store the model name `""Database cleanup test field-model"` in a class attribute to not copy it.

Reply:
Hi, the model `Database cleanup test field-model` is added inside the new env `with environment() as env:` (at the line 22). It will not be deleted after the test.

So, i have to delete it in `def tearDownClass(self)`, otherwise i will face an error when running the test from the second time:

```
bad query: b'INSERT INTO "ir_model" ("create_date", "create_uid", "model", "name", "order", "state", "write_date", "write_uid") VALUES (\'2024-10-10 03:29:45.693857\', 1, \'x_database.cleanup.test.field.model\', \'{"en_US": "Database cleanup test field-model"}\', \'id\', \'manual\', \'2024-10-10 03:29:45.693857\', 1) RETURNING "id"'
ERROR: duplicate key value violates unique constraint "ir_model_obj_name_uniq"
DETAIL:  Key (model)=(x_database.cleanup.test.field.model) already exists.
```